alpha/action/render.go

```diff
@@ -56,12 +56,16 @@ type Render struct {
 	skipSqliteDeprecationLog bool
 }
 
-func nullLogger() *logrus.Entry {
+func NullLogger() *logrus.Entry {
 	logger := logrus.New()
 	logger.SetOutput(ioutil.Discard)
 	return logrus.NewEntry(logger)
 }
 
+func NewRenderCacheDir() (string, error) {
+	return os.MkdirTemp("", "render-registry-")
+}
+
```

Comment:
IMO, we should keep the `NullLogger` and `NewRenderCacheDir` functions out of the exported API of this library.

For such a small amount of code, I would lean toward duplicating these functions in the render CLI code. That would basically mean that the default CLI registry would be decoupled from the default registry created by the action.Render library, and to be honest, I don't see that as a bad thing.

If we think its valuable enough to share this code AND avoid duplication, we could make an `internal/` tree of packages that can only be used by other packages within this repo. e.g. `internal/logutil.NullLogger()` and `internal/renderutil.NewCacheDir()`.

Reply:
I agree that duplicating the code is probably the best way to go. The main reason that I created the `NewRenderCacheDir` was so that the same tmp directory prefix would be used, but I think unless there's another place this is required it's fine just to duplicate the string. I've pushed the changes.